### DISCUSSION TITLE
Lilygo T8-C3V1 and teensy41

### DISCUSSION THREAD

**Discussion by mghaff007** [forkineye/ESPixelStick#614] [General]
Does anyone know if there is a place with a write up on how to add cards or boards to the PlatformIO files? I'm asking cause I was going to try and place ESPixelStick on a Teensy4.1 but also have some lilygo -C3 boards I would like to attempt to get pixelstick to work on. Any advice would be greatly appreciated!  FYI when I first came across this project 3 years ago It was what got me into Christmas lights again so Thanks! :)

**Comment by MartinMueller2003** [comment]
There is no writeup. For a new platform, you need to modify the Espixelstick project and the Flashtool project.

**Comment by MartinMueller2003** [comment]
You wont be able to use the code as is on any non ESP platforms. The code assumes the ESP CPU architecture. You could try to adapt the OutputUart files to a different type of device but you will run into RAM issues pretty quickly.

The LilyGo boards would not be a problem. You can look at the files in the platforms directory and make one for your platform. Post it here and I will add it to the supported platforms.
